src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/CreateGeneralLetterConsentMidHandler.java

```diff
@@ -49,10 +49,10 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
         log.info("Received request to consent general letter for Case ID: {}", caseDetails.getId());
         validateCaseData(callbackRequest);
         FinremCaseData finremCaseData = caseDetails.getData();
-        finremCaseData.setIntervenerOneWrapper(null);
-        finremCaseData.setIntervenerTwoWrapper(null);
-        finremCaseData.setIntervenerThreeWrapper(null);
-        finremCaseData.setIntervenerFourWrapper(null);
+        finremCaseData.setIntervenerOne(null);
```

Comment:
We have FirnremCaseDataTest unit test which checks for wrapper, I recon to keep the same name wrapper

Reply:
this is not a wrapper hence why I changed the name